### ISSUE TITLE
DCAE Hyperparameter optimization

### ISSUE THREAD

**Issue by vrodriguezf** [pacmel/timecluster_hub#3]
In GitLab by @vrodriguezf on Mar 18, 2020, 18:14

According to the paper: The number of feature maps, size
of filter and depth of the model are set based on the recon-
struction error on validation set.

Use weights & biases to analyse which hyperparameters are best for the mining data.
